CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 **Sample app**
 
 - **Feature:** Display of a banner on message contact picker when required
+- **Feature:** Added Points Interface `getPoints` to retrieve Rakuten Points
+- **Feature:** Added `rakuten.miniapp.user.POINTS` custom permission
```

Comment:
These two items should go under the "SDK" section above.

Reply:
yeah true just changed it 👍